soes/esc.c

```diff
@@ -1143,5 +1123,5 @@ void ESC_state (void)
  */
 void ESC_config (esc_cfg_t * cfg)
 {
-   esc_cfg = cfg;
+   ESCvar.esc_cfg = cfg;
```

Comment:
It may be better to copy the contents of cfg into ESCvar, in case the user's variable goes out of scope. Otherwise there should be a comment saying that the config must not go out of scope.

If we copy the contents into ESCvar it may be possible to simplify the ESCvar structure (e.g. maybe have ESCVar.pdosm instead of ESCvar.esc_cfg.pdosm).